modules/replicator/build.gradle

```diff
@@ -29,6 +29,9 @@ dependencies {
     implementation project(':ignite-core')
     implementation project(':ignite-raft')
     implementation project(':ignite-raft-api')
+    implementation project(':ignite-metastorage-api')
```

Comment:
Does really needed?

Reply:
Without this I can't pass `MetaStorageService` into `startReplica` :< It's required by `RebalanceRaftGroupEventsListener`'s constructor